kasa/cli/discover.py

```diff
@@ -93,13 +145,37 @@ async def print_discovered(dev: Device):
     for device in discovered_devices.values():
         await device.protocol.close()
 
-    echo(f"Found {len(discovered)} devices")
-    if unsupported:
-        echo(f"Found {len(unsupported)} unsupported devices")
-    if auth_failed:
-        echo(f"Found {len(auth_failed)} devices that failed to authenticate")
 
-    return discovered
+@discover.command()
+@click.pass_context
+async def config(ctx):
```

Comment:
This should probably error out if `--target` is defined:
```
kasa --target 192.168.x.255 discover config
[bold red]Unable to connect to None[/bold red]
```
Or maybe this doesn't even belong to the discovery but elsewhere?

Reply:
Check added to main `cli` command.